docs/use-linea/explore/use-vitidiary.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Use VitiDiary for Linea Voyage Quest
```

Comment:
Can you remove `for Linea Voyage Quest`

Didn't notice it for the attic tutorial, but you can remove that from the title as well in this PR?

Reply:
removed both now